manifests/g/git-cola/git-cola/4.6.1/git-cola.git-cola.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: git-cola.git-cola
+PackageVersion: 4.6.1
+InstallerType: zip
+NestedInstallerType: nullsoft
+NestedInstallerFiles:
+- RelativeFilePath: git-cola-4.6.1.windows/git-cola-4.6.1.exe

```

Comment:
The nested installer appears to be in the root of the `extracted` folder:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/93534366-1903-4b7b-b904-3d234c909f69)

```suggestion
- RelativeFilePath: git-cola-4.6.1.exe
```